Parse/Internal/Command/ParseCommand.cs

```diff
@@ -46,14 +46,13 @@ internal class ParseCommand : HttpRequest {
         IList<KeyValuePair<string, string>> headers = null,
         Stream stream = null,
         string contentType = null) {
-      Uri = new Uri(ParseClient.HostName, relativeUri);
+      Uri = new Uri(ParseClient.HostName + relativeUri);
```

Comment:
Personally, I dislike this idea of doing it this way, because this requires that either:

 1. HostName ends with a `/`
 2. or relativeUri begins with a `/`

But never both. It works, I'm just afraid of something weird happening here as we move toward 'HostName' being more public.